packages/strapi-plugin-users-permissions/controllers/User.js

```diff
@@ -35,6 +35,11 @@ module.exports = {
 
   me: async (ctx) => {
     const user = ctx.state.user;
+
+    if (!user) {
+      return ctx.badRequest(null, [{ messages: [{ id: 'No authorization header was found' }] }]);
```

Comment:
Have to use `ctx.unauthorized` to send a 401 response code nop?
It will never match ` if (response.status === 401) {` with `ctx.badRequest `.